topics/admin/tutorials/data-library/tutorial.md

```diff
@@ -0,0 +1,269 @@
+
+> ### {% icon hands_on %} Hands-on: Setting up Data Libraries
+>
+> 1. We will add a pre-task to clone [a data repository](https://github.com/galaxyproject/galaxy-test-data) into your machine. We will use this as the source for a library dataset.
+>
+>    ```diff
+>    --- a/galaxy.yml
+>    +++ b/galaxy.yml
+>    @@ -5,6 +5,9 @@
+>         - name: Install Dependencies
+>           package:
+>             name: ['git', 'make', 'python3-psycopg2', 'virtualenv', 'tar', 'bzip2']
+>    +    - git:
+>    +        repo: 'https://github.com/usegalaxy-eu/libraries-training-repo'
+>    +        dest: /libraries/
+>       handlers:
+>         - name: Restart Galaxy
+>           systemd:
+>    ```
+>
+> 4. Edit the file `group_vars/galaxyservers.yml` and set the following variables:
+>
+>    ```diff
+>    --- a/group_vars/galaxyservers.yml
+>    +++ b/group_vars/galaxyservers.yml
+>    @@ -28,6 +28,9 @@ miniconda_manage_dependencies: false
+>
+>     galaxy_config:
+>       galaxy:
+>    +    library_import_dir: /libraries/admin
+>    +    user_library_import_dir: /libraries/user
+>         job_resource_params_file: "{{ galaxy_config_dir }}/job_resource_params_conf.xml"
+>         tool_destinations_config_file: "{{ galaxy_config_dir }}/tool_destinations.yml"
+>    ```
+>
+> 5. Run the playbook:
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```
+>    > ansible-playbook galaxy.yml
+>    > ```
+>    {: .code-in}
+>
+{: .hands_on}
+
+# Importing Data
+
+There are multiple options for importing data from your server, we'll go through all of your choices below. But first, let's take a quick look at the example library structure we've provided.
+
+> > ### {% icon code-in %} Input: Bash
+> > ```bash
+> > tree /libraries
+> > ```
+> {: .code-in}
+>
+> > ### {% icon code-out %} Output: Bash
+> > ```
+> > /libraries/
+> > ├── admin
+> > │   └── admin-wildtype.fna
+> > ├── example-library.yaml
+> > ├── README.md
+> > └── user
+> >     ├── admin@example.com
+> >     │   └── user-wildtype.fna
+> >     └── admin@example.org
+> >         └── user-wildtype.fna
+> >
+> > 4 directories, 5 files
+> > ```
+> {: .code-out}
+{: .code-2col}
+
+We have a directory named `admin`, which will be available to all admin users (we set `library_import_dir: /libraries/admin` earlier.)
+
+Additionally we have a `user` directory, below the user directory are more directories with the user's email as they directory key. Data can be placed in here, and it will become accessible to those users (we set `user_library_import_dir: /libraries/user` for this.)
+
+![An add datasets dropdown menu in galaxy showing the options from history, from user directory, and under admins only, from import directory](../../images/data/import-menu.png)
+
+## from History
+
+This is by far the easiest and most convenient option for small datasets, or datasets that are just already in a history
+
+![A select box is shown listing files in the user's history](../../images/data/import-admin.png)
```

Comment:
Maybe zoom to show only the box or reduce the browser window width? Some images is very large making it hard to read without clicking on the image

Reply:
Cropped :)